src/client/ClientGameRunner.ts

```
@@ -336,6 +348,45 @@
+      event.y,
+    );
+    if (!this.gameView.isValidCoord(cell.x, cell.y)) {
+      return;
+    }
+    const tile = this.gameView.ref(cell.x, cell.y);
+    this.myPlayer.actions(tile).then((actions) => {
+      if (this.lastSelectedBuildableUnit !== null) {
+        const unit = actions.buildableUnits.find(
+          (u) => u.type === this.lastSelectedBuildableUnit,
+        );
+        if (unit && unit.canBuild) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)